frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -14,6 +15,9 @@ export const UnsubscribeSurveyModal = ({ product }: { product: BillingProductV2T
         <LemonModal
             onClose={() => {
                 setSurveyID('')
+                posthog.capture('survey dismissed', {
+                    $survey_id: surveyID,
+                })
```

Comment:
just making sure.. does this trigger on any close of the modal, or just when someone uses the x button? If it's any close of the modal then we'd be sending the event even when it just goes away because someone submitted. If not then all good!

Reply:
Good point. I just confirmed that these 'survey dismissed' events are not getting triggered on submit.